src/components/app/MenuButtons/Publish.js

```diff
@@ -200,12 +203,12 @@ class MenuButtonsPublishImpl extends React.PureComponent<PublishProps> {
             Publishing profile…
           </div>
           <div className="menuButtonsPublishUploadPercentage">
-            {uploadProgress}
+            {uploadProgressString}
           </div>
           <div className="menuButtonsPublishUploadBar">
             <div
               className="menuButtonsPublishUploadBarInner"
-              style={{ width: uploadProgress }}
+              style={{ width: `${uploadProgress * 100}%` }}
```

Comment:
Testing the PR, I realize that we never see this, because the panel closes down when we start publishing. Is it expected or is it another issue?

Reply:
It'll still show when you open it back up I believe. The original implementation didn't close the panel.

Reply:
Nope you can't show it back because the "publish" button turns into a "cancel" button. :)